library/lib/application.cpp

```diff
@@ -132,6 +132,64 @@ static void windowKeyCallback(GLFWwindow* window, int key, int scancode, int act
     }
 }
 
+struct CursorPos {
```

Comment:
Not sure if we want that dumped in the cpp like that :eyes:

Reply:
Yea I was unsure myself. I mean I see no negative removing it and just using the two doubles by themselves. I'll be honest and just say I was feeling lazy and wanted to save on typing.

It's your call, I can tidy things up by removing it or placing it somewhere else.

Reply:
I mean all this stuff here, all the variables, why are they not Application class members?

Reply:
That is a good point, they should be Application class members.

On it captain!